serviceaccounts.rb

```diff
@@ -9,12 +9,16 @@
 # The test SA key is a special case for local development, and is initialized
 # but not deleted.
 class ServiceAccountContext
+  attr_reader :service_account
 
   SERVICE_ACCOUNT_KEY_PATH = "sa-key.json"
 
   def initialize(project, service_account = nil, path = nil)
     @project = project
     @service_account = service_account
+    if not service_account
+      @service_account = "#{@project}@appspot.gserviceaccount.com"
+    end
     @path = path
```

Comment:
nit: can we name this `@credentials_keyfile_path` or something specific?

Reply:
changed to keyfile_path